dataprofiler/data_readers/json_data.py

```diff
@@ -23,14 +25,17 @@ def __init__(self, input_file_path=None, data=None, options=None):
         Possible Options::
         
             options = dict(
-                data_format= type: str, choices: "dataframe", "records", "json"
+                data_format= type: str, choices: "dataframe", "records", "json",
+                 "data_stream"
```

Comment:
don't we not have "data_stream" any more?

Reply:
swapped it for flattened dataframe, somewhat more apt